ui/apps/platform/src/Containers/VulnMgmt/List/Images/VulnMgmtListImages.js

```diff
@@ -180,44 +180,19 @@ export function getCurriedImageTableColumns(watchedImagesTrigger, isFeatureFlagE
                 sortable: false,
             },
             {
-                Header: `Deployments`,
+                Header: `Entities`,
```

Comment:
Just to confirm, we don't need to feature flag the column changes here, right?

Reply:
We aren't feature-flagging the column itself, just whether it shows Components or Image Components.